CMakeLists.txt

```diff
@@ -1,17 +1,45 @@
 cmake_minimum_required(VERSION 2.8 FATAL_ERROR)
 
+set(ENABLE_TEST true)
+set(USE_DART true)
+
 find_package(gazebo 8.0 REQUIRED)
 
-include_directories(${PROJECT_SOURCE_DIR} ${GAZEBO_INCLUDE_DIRS})
-link_directories(${GAZEBO_LIBRARY_DIRS})
-list(APPEND CMAKE_CXX_FLAGS "${GAZEBO_CXX_FLAGS}")
-  
+# DART dependencies are not automatically exported by Gazebo cmake script,
+# need to find it separately to not get undefined references to gazebo libs
+# compiled with DART support.
+if(USE_DART)
+  find_package(DART REQUIRED)
+endif(USE_DART)
```

Comment:
I think we should try to fix this upstream in gazebo's cmake config file. Maybe we can address it in the dart-6 branch?